server/utils/utils.ts

```diff
@@ -126,3 +128,9 @@ export const govukSelectSetSelected = (items: GovukSelectItem[], value: string):
     selected: 'value' in entry ? entry.value === value : entry.text === value,
   }))
 }
+
+const template = '{% from "govuk/components/input/macro.njk" import govukInput %}{{ govukInput(params) }}'
+
+export default function freeTextInput(params: object) {
+  return nunjucks.renderString(template, { params })
+}
```

Comment:
what’s this utility for?

Reply:
So this is used to help add the conditional input entries for radio buttons. I was playing with it to get it working, but I didn't quite get there as it wasn't showing. So I left it for now to get other things done, but I will come back to this and likely use this function when I return to the other page designs.
I can remove it for now if you would like and can then re-add it when I go to add those conditional fields.

Reply:
your call, i guess! i can't tell quite how it'll work 🤷